linux_os/guide/services/ssh/ssh_server/sshd_enable_strictmodes/oval/shared.xml

```diff
@@ -16,27 +17,47 @@ and configurations.</description>
       <criteria comment="sshd is not installed" operator="AND">
         <extend_definition comment="sshd is not required or requirement is unset"
         definition_ref="sshd_not_required_or_unset" />
+        {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+        <extend_definition comment="rpm package openssh removed"
+        definition_ref="package_openssh_removed" />
+        {{% else %}}
         <extend_definition comment="rpm package openssh-server removed"
         definition_ref="package_openssh-server_removed" />
+        {{% endif %}}
       </criteria>
       <criteria comment="sshd is installed and configured" operator="AND">
         <extend_definition comment="sshd is required or requirement is unset"
         definition_ref="sshd_required_or_unset" />
+        {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+        <extend_definition comment="rpm package openssh installed"
+        definition_ref="package_openssh_installed" />
+        {{% else %}}
         <extend_definition comment="rpm package openssh-server installed"
         definition_ref="package_openssh-server_installed" />
+        {{% endif %}}
         <criterion comment="Check StrictModes in /etc/ssh/sshd_config"
         test_ref="test_sshd_enable_strictmodes" />
       </criteria>
     </criteria>
   </definition>
+{{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="tests the value of StrictModes setting in the /etc/ssh/sshd_config file"
+  id="test_sshd_enable_strictmodes" version="1">
+{{% else %}}
   <ind:textfilecontent54_test check="all" check_existence="none_exist"
   comment="tests the value of StrictModes setting in the /etc/ssh/sshd_config file"
   id="test_sshd_enable_strictmodes" version="1">
+{{% endif %}}
     <ind:object object_ref="obj_sshd_enable_strictmodes" />
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sshd_enable_strictmodes" version="1">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
+{{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+    <ind:pattern operation="pattern match">^[\s]*(?i)StrictModes(?-i)[\s]+yes[\s]*(?:|(?:#.*))?$</ind:pattern>
+{{% else %}}
     <ind:pattern operation="pattern match">^[\s]*(?i)StrictModes(?-i)[\s]+no[\s]*(?:|(?:#.*))?$</ind:pattern>
+{{% endif %}}
```

Comment:
Looks like having StrictModes set to `no` is a bug. Go ahead and remove the SuSE customization here and set to `yes`